message-me/app/javascript/packs/application.js

```diff
@@ -12,9 +12,19 @@ import '@doabit/semantic-ui-sass'
 Rails.start()
 Turbolinks.start()
 ActiveStorage.start()
+
+window.scroll_bottom = function(id) {
```

Comment:
```suggestion
scroll_bottom = function(id) {
```

This should be sufficient.